### DISCUSSION TITLE
Seeking web developers with disabilities to interview for feedback

### DISCUSSION THREAD

**Discussion by erdraud** [community/community#27396] [Accessibility]
Hi, y'all! I'm a PM on the[ Microsoft Edge Developer Tools](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/overview) team, and I'm looking for 1-3 web developers with disabilities that use the accessibility features within DevTools or use assistive technology to interact with DevTools so that we can get an in-depth understanding of your workflows and major pain points with our tooling. We'd want to set aside some time to connect with you to get a baseline idea of pain points then follow up later to get your feedback on the solutions we create. Thanks!

**Comment by cehfisher** [comment]
Hi @erdraud - I hope you were able to get the testers with lived experience you needed for your research study. If you could please close this issue as resolved, that would be great. Thanks! (cc @queenofcorgis)
